.agent/skills/basic-documentation/SKILL.md

```
@@ -1,499 +1,126 @@
 name: basic-documentation
 description: Documentation standards - README structure, ROADMAP patterns, skill documentation format (YAML frontmatter), markdown conventions, security documentation, API docs, and best practices. Guidelines for creating and maintaining all project documentation. Keywords: documentation, readme, roadmap, markdown, documentation-standards, yaml-frontmatter, technical-writing, api-documentation, changelog
+last-updated: 2026-02-15
+difficulty: basic
 ---
 
 # Documentation
 
-This skill defines the standards for creating and maintaining documentation within the DRN-Project ecosystem. It encapsulates patterns for root-level documents (README, ROADMAP, CHANGELOG), agent skills, API documentation, and security documentation.
+Standards for creating and maintaining documentation within DRN-Project.
 
 ## 1. Core Principles (DiSCOS)
```

markdownlint-cli2 (0.20.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)